content/agent/autodiscovery.md

```diff
@@ -134,7 +134,7 @@ In the `datadog.yaml` file, set the `<KV_STORE_IP>` address and `<KV_STORE_PORT>
 
 ```
 # The providers the Agent should call to collect checks configurations.
-# Note that the File Configuration Provider is enabled by default and cannot
+# Please note the File Configuration Provider is enabled by default and cannot
```

Comment:
We don't [say please](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md). 😄